solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -1490,7 +1490,7 @@ parameter_types! {
     pub const DepositAmount: Balance = 100 * UNITS;
     #[derive(Clone)]
     pub const MaxLengthParaIds: u32 = 100u32;
-    pub const MaxEncodedGenesisDataSize: u32 = 5_000_000u32; // 5MB
+    pub const MaxEncodedGenesisDataSize: u32 = 1_000_000u32; // 1MB
```

Comment:
Maybe add a comment that this should match `max_head_data_size` from the relay chain

Reply:
_fixed_